lib/CMakeLists.txt

```diff
@@ -51,12 +51,13 @@ endif()
 
 # The rest of the build
 
-include_directories(${CMAKE_CURRENT_BINARY_DIR}/../include)
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/..)
-include_directories(${CMAKE_CURRENT_SOURCE_DIR}/../include)
-include_directories(${CMAKE_CURRENT_BINARY_DIR}/..)
-include_directories(${CMAKE_CURRENT_SOURCE_DIR})
-include_directories(${CMAKE_CURRENT_BINARY_DIR})
+include_directories(
+  ${CURL_BINARY_DIR}/include/curl # for generated curlbuild.h from curl.h
```

Comment:
is there a reason for the change of variable name?
`${CMAKE_CURRENT_XXX_DIR}` are standard names that will be familiar to anyone used to cmake. I think they should be prefered unless there's a good reason not to.